src/test/java/com/gooddata/md/MetadataServiceTest.java

```diff
@@ -96,7 +96,8 @@ public void testUpdateObj() throws Exception {
         final Updatable resultObj = mock(Updatable.class);
 
         when(obj.getUri()).thenReturn(URI);
-        when(restTemplate.getForObject(URI, obj.getClass())).thenReturn(resultObj);
+        final Updatable forObject = restTemplate.getForObject(URI, obj.getClass());
```

Comment:
this is unfortunately not possible, since that way the mocked `getForObject` wouldn't match